sdk/lib/atl/atlimage.h

```diff
@@ -1016,6 +1017,7 @@ class CImage
     HBITMAP             m_hBitmap;
     mutable HBITMAP     m_hOldBitmap;
     mutable HDC         m_hDC;
+    mutable LONG        m_nDCRefCount = 0;
```

Comment:
Why some of these members need the "mutable" keyword? Is it because they get modified by some functions marked as "const" ?

Reply:
> Is it because they get modified by some functions marked as "const" ?

Yep. Even `const` function can change the mutable variables.